config/locales/simple_form.uk.yml

```diff
@@ -0,0 +1,12 @@
+uk:
+  simple_form:
```

Comment:
no :) Kostya "uk" is official ukrainian shortcut
https://github.com/svenfuchs/rails-i18n/blob/master/rails/locale/uk.yml